src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -684,13 +683,14 @@ event::ptr primitive_inst::realloc_if_needed() {
             prealloc_shape[seq_axis] += tmp_prealloc_count;
             required_buffer_size = std::accumulate(prealloc_shape.begin(), prealloc_shape.end(), size_t(1), std::multiplies<size_t>());
         } else {
-            required_buffer_size = (updated_layouts[i].get_buffer_size().count());
+            required_buffer_size = (updated_layouts[i].padded_count());
         }
         if (required_buffer_size * 10 < _max_output_layout_count[i]) {
             reclaim = true;
         }
+        reclaim = false;
```

Comment:
Looks like some experimental change. Is it intended?

Reply:
oh no, reverted